docs/src/apimanual.md

```diff
@@ -565,4 +565,39 @@ If `canaddconstraint` returns `false`, then calling `addconstraint!` must throw
 
 ### Package Naming
 
-MOI solver interfaces may be in the same package as the solver itself (either the C wrapper if the solver is accessible through C, or the Julia code if the solver is written in Julia, for example). In some cases it may be more appropriate to host the MOI wrapper in its own package; in this case it is recommended that the MOI wrapper package be named `MathOptInterfaceXXX` where `XXX` is the solver name.
+MOI solver interfaces may be in the same package as the solver itself (either the C wrapper if the solver is accessible through C, or the Julia code if the solver is written in Julia, for example).
+The guideline for naming the file containing the MOI wrapper is `src/MOIWrapper.jl` and `test/MOIWrapper.jl` for the tests.
+In some cases it may be more appropriate to host the MOI wrapper in its own package; in this case it is recommended that the MOI wrapper package be named `MathOptInterfaceXXX` where `XXX` is the solver name.
+
+### Testing guideline
+
+The skeleton below can be used for the wrapper test file of a solver name `FooBar`:
+```julia
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIT = MOI.Test
+const MOIU = MOI.Utilities
+const MOIB = MOI.Bridges
+
+# Include here the functions/sets supported by the solver wrapper (not those that are supported through bridges)
+MOIU.@model FooBarModelData () (EqualTo, GreaterThan, LessThan) (Zeros, Nonnegatives, Nonpositives) () (SingleVariable,) (ScalarAffineFunction,) (VectorOfVariables,) (VectorAffineFunction,)
```

Comment:
Could the UniversalFallback work here? There's no performance concerns for testing.

Reply:
Universal Fallback still requires an internal model so we need to create a model anyway. Doing `MOIU.@model FooBarModelData () () () () () () () ()` and then using UniversalFallback would be a bit obscure. Besides, UniversalFallback only do attribute fallback for now :)